logging/wagclientlogger/wagclientlogger.go

```diff
@@ -0,0 +1,23 @@
+package wagclientlogger
+
+//WagClientLogger provides a minimal interface for a Wag Client Logger
+type WagClientLogger interface {
```

Comment:
Out of curiosity, is there a spec for the new logging interfaces (both client and server)? I'd be interested in reading it.

Reply:
It was briefly mentioned in the original wagv9 spec by Taylor. There is no new logging interface for the server (but it is a change we want to eventually make). The idea was just a minimal wrapper that can be easily applied to any logger but similar to kayvee-go. We plan on using kayvee-go for the foreseeable future but don't want to have to import it into the client. 

It will eventually be totally obsoleted by [this](https://github.com/golang/go/discussions/54763) hopefully.